modules/process-testing/pages/bonita-test-extension.adoc

```diff
@@ -0,0 +1,36 @@
+= The Bonita test extension
+:description: Discover the Bonita test extension, a JUnit 5 extension that parametrize the Bonita client used in integration tests.
+
+The Bonita test extension is the entry point to create Bonita integration tests. It allows to specify some behaviors, and to inject the Bonita Test Toolkit in the test methods.
+
+[NOTE]
+====
+Integrations tests are built upon the Bonita Test Toolkit, based on the open source https://github.com/bonitasoft/bonita-java-client[Bonita Java Client]. +
+The Bonita Test Toolkit is only available for Enterprise, Performance, Efficiency and Teamwork editions. 
+====
+
+== Create and configure the Bonita test extension
+
+The Bonita test extension is a JUnit 5 extension, it has to be registered using the annotation `org.junit.jupiter.api.extension.RegisterExtension`.
+
+The following elements can be configured: 
+
+**- Target runtime URL**: The targeted Bonita runtime URL. It is used to perform HTTP requests to interact with this runtime.
+
+**- Delete process instances**: Whether all existing process instances (including archived instances and their documents) should be deleted before tests execution.
+
+**- Clear BDM**: Whether the BDM database of the targeted Bonita should be cleared before tests execution. All existing business data will be removed.
+
+[source, Java]
+----
+ @RegisterExtension <1>
```

Comment:
Prefer the `@BonitaTests` annotation

You may document both way.

Reply:
Hum OK. I think I'll also document the way to configure test container in this page.

Reply:
I don't think we should document testcontainers as the goal is to use it with BCD on  a provisionned environment

Reply:
As you wish, but in my view the best solution is to use test container and the builder and the deployer. It's sad to see that we do not want to provide our users the best tooling to use.